extensions/rich_text_components/components.py

```diff
@@ -142,7 +142,16 @@ class Link(BaseRteComponent):
 class Math(BaseRteComponent):
     """Class for Math component."""
 
-    pass
+    @classmethod
+    def validate(cls, value_dict):
+        """Validates Math component."""
+        super(Math, cls).validate(value_dict)
+        filename_pattern_regex = (
+            r'mathImg_[0-9]+_\S{10}_height_[0-9d]+_width_[0-9d]+_vertical_['
+            '0-9d]+.(svg)$')
+        filename = value_dict['math_content-with-value']['svg_filename']
+        if not re.match(filename_pattern_regex, filename):
+            raise Exception('Invalid filename')
```

Comment:
`Invalid svg_filename attribute in math component:  %s`

Reply:
done